src/style.css

```diff
@@ -1 +1,49 @@
-/* add styles here */
+body {
+  font-family: Arial, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0;
+  padding: 20px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('images/star.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-right: 8px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url('images/star-active.svg');
+}
```

Comment:
The selector '.stars--0 .stars__star:nth-child(-n + 0)' will not apply any styles because '-n + 0' does not select any elements. If the intention is to have no active stars for '.stars--0', this rule can be removed or adjusted to reflect the intended design.